src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -283,20 +283,44 @@
     "themes": [
         {
             "name": "light",
-            "window":{
+            "window":
+            {
                 "applicationTheme": "light"
+            },
+            "tab":
+            {
+                "background": "terminalBackground",
+                "unfocusedBackground": "#00000000"

```

Comment:
Just alpha 0 so it blends in with the rest of the tab bar?

Reply:
yeppers. Basically cause our tabs RN are already "transparent when unfocused". This gives them "terminalBG" when focused, but retains "transparent when unfocused", so that the black of campbell doesn't show up semi-transparent on unfocused tabs even in light mode.